src/app/dev.js

```diff
@@ -14,90 +15,89 @@ import { getModuleContext, getChanged } from './util/webpack';
  * Register all templates beforehand with a require context, and save the partial by filename
  */
 const [partialModules, partialsContext] = getModuleContext(
-	require.context('./component/blocks/', true, /\.hbs$/),
-	(context, key, module) => {
-		Handlebars.registerPartial(/\/([^/]+)\.hbs/gi.exec(key)[1], module)
-	}
+  require.context('./component/blocks/', true, /\.hbs$/),
+  (context, key, module) => {
+    Handlebars.registerPartial(/\/([^/]+)\.hbs/gi.exec(key)[1], module);
+  },
 );
 
 /**
  * json data context, for hot reloading
  */
-const [jsonModules, jsonContext] = getModuleContext(
-	require.context('../data/', true, /\.json$/),
-);
+const [jsonModules, jsonContext] = getModuleContext(require.context('../data/', true, /\.json$/));
 
 // Get info for current page
 const pageMatch = /\/(.*)\.html/i.exec(document.location.pathname);
-const jsonModuleName = (pageMatch && pageMatch[1] || 'index');
-const getJsonData = () => {
-	return jsonModules[`./${jsonModuleName}.json`];
-};
-
+const jsonModuleName = (pageMatch && pageMatch[1]) || 'index';
+const getJsonData = () => jsonModules[`./${jsonModuleName}.json`];
 
 function render() {
-	const div = document.getElementById('app');
+  const div = document.getElementById('app');
 
-	if (jsonModuleName === 'index') {
-		// render the index overview page
-		div.innerHTML = indexTemplate({
-			pages: Object.keys(jsonModules)
-				.map(key => ({
-					page: key.split('/').pop().split('.').slice(0, -1).join('.'),
-					data: jsonModules[key],
-				}))
-				.sort()
-				.map(({ page, data }) => ({
-					page,
-					link: page + '.html',
-					data,
-				}))
-		});
-	} else {
-		// render page with data
-		div.innerHTML = appTemplate(getJsonData());
+  if (jsonModuleName === 'index') {
+    // render the index overview page
+    div.innerHTML = indexTemplate({
+      pages: Object.keys(jsonModules)
+        .map(key => ({
+          page: key
+            .split('/')
+            .pop()
+            .split('.')
+            .slice(0, -1)
+            .join('.'),
+          data: jsonModules[key],
+        }))
+        .sort()
+        .map(({ page, data }) => ({
+          page,
+          link: `${page}.html`,
+          data,
+        })),
+    });
+  } else {
+    // render page with data
+    div.innerHTML = appTemplate(getJsonData());
 
-		// TODO: should we cleanup ALL block JS modules?
-		initComponents(div);
-	}
+    // TODO: should we cleanup ALL block JS modules?
+    initComponents(div);
+  }
 }
-document.addEventListener("DOMContentLoaded", () => {
-	render();
+document.addEventListener('DOMContentLoaded', () => {
+  render();
 });
 
-
 // Hot Module Replacement API
 if (module.hot) {
-	module.hot.accept(['./component/layout/index/index.hbs', './component/layout/app/app.hbs'], () => {
-		render();
-	});
+  module.hot.accept(
+    ['./component/layout/index/index.hbs', './component/layout/app/app.hbs'],
+    () => {
```

Comment:
`() => render(),`

Reply:
Technically the function doesn't return anything, so using a function body is more correct IMO.

Reply:
Agreed.